src/app/pages/apps/components/app-detail-view/app-detail-view.component.html

```diff
@@ -35,7 +35,7 @@
           color="primary"
           [ixTest]="[app?.name, 'install']"
           [disabled]="!app"
-          [routerLink]="['/apps', 'available', appId, 'install']"
```

Comment:
`routerLink` needs to be added back with new params

Reply:
The method `navigateToInstallPage` handles navigation now on click